poll_suggestion/tests.py

```diff
@@ -1,3 +1,82 @@
 # from django.test import TestCase
 
 # Create your tests here.
+# from django.test import TestCase
+
+# Create your tests here.
+from poll.models import Poll, get_default_end_date
+from .models import PollSuggestion
+from datetime import datetime, timedelta
+from django.utils import timezone
+from event.models import Event
+from category.models import Category
+from location.models import Location
+from category_location.models import CategoryLocation
+import pytest
+
+EVENT_NAME = "weight lifting"
+CATEGORY = "Gym1"
+LOCATION = "Holmes place Netanya"
+MAX_PART = 10
+IS_PRIVATE = False
+DATETIME = datetime.now()
+MAX_SUGEGSTIONS = 5
+DATETIME = timezone.now()
+
+
+@pytest.fixture
+def setup_category_location():
+    loc = Location(
+        name=LOCATION,
+        city="test city",
+        street="test street",
+        street_number=1,
+        indoor=False,
+        description="test description",
+    )
+    loc.save()
+    cat = Category(name=CATEGORY)
+    cat.save()
+    cat_loc = CategoryLocation(location=loc, category=cat)
+    cat_loc.save()
+    return cat_loc
+
+
+@pytest.fixture
+def setup_event(setup_category_location):
+    event = Event(
+        category=setup_category_location.category,
+        location=setup_category_location.location,
+        poll=None,
+        name=EVENT_NAME,
+        max_participants=MAX_PART,
+        start_time=DATETIME + timedelta(days=2),
+        end_time=DATETIME + timedelta(days=3),
+        is_private=IS_PRIVATE,
+    )
+    event.save()
+    return event
+
+
+@pytest.fixture
+def setup_poll(setup_event):
+    poll = Poll(event_id=setup_event, max_suggestions=MAX_SUGEGSTIONS, end_time=get_default_end_date())
+    poll.save()
+    return poll
+
+
+@pytest.fixture
+def set_up_poll_suggestion(setup_poll):
+    poll_s = PollSuggestion.objects.create(time=datetime.now() + timedelta(days=3), poll_id=setup_poll)
```

Comment:
in continuous to the previous comment
```suggestion
1.    poll_s = PollSuggestion.objects.create(time=datetime.now() + timedelta(days=3), poll_id=setup_poll)
 OR
2.    poll_s = PollSuggestion.objects.create(time=datetime.utcnow() + timedelta(days=3), poll_id=setup_poll)
```